src/components/InformationScreen.tsx

```diff
@@ -1,18 +1,59 @@
 import React from 'react';
-import { View } from 'react-native';
-import { Text } from 'react-native-paper';
+import { View, FlatList } from 'react-native';
+import { Text, Card, Title} from 'react-native-paper';
 
 import { createStackNavigator } from '@react-navigation/stack';
 import { CustomNavigationBar } from './CustomNavigationBar';
 import { SettingsScreen } from './SettingsScreen';
+import { ActivityInfo } from './info/ActivityInfo';
+import { CalenderInfo } from './info/CalenderInfo';
+import { ValuesInfo } from './info/ValuesInfo';
+import { BehaviouralActivation } from './info/BehaviouralActivation';
 
 const InformationStack = createStackNavigator();
 
-const ViewContent = () => {
+const cardsData = [
+  {
+    title: "BehaviouralActivation",
+    imageUri: "https://picsum.photos/700",
+    navigateTo: "BehaviouralActivation",
+  },
+  {
+    title: "The Activityscreen",
+    imageUri: "https://picsum.photos/700",
+    navigateTo: "ActivityInfo",
+  },
+  {
+    title: "The Valuesscreen",
+    imageUri: "https://picsum.photos/700",
+    navigateTo: "ValuesInfo",
+  },
+  {
+    title: "The Calendarscreen",
+    imageUri: "https://picsum.photos/700",
+    navigateTo: "CalenderInfo",
+  },
+];
+
+const ViewContent = ({navigation}: any) => {
+  const renderItem = ({item}:any) => (
+    <Card
+    onPress={() => navigation.navigate(item.navigateTo)}
+    style={{marginBottom: 20, padding: 10}}>
+        <Card.Content style={{alignItems: 'center'}}>
+          <Title>{item.title}</Title>
+        </Card.Content>
+        <Card.Cover source={{uri: item.imageUri}} />
+      </Card>
+  );
+
   return (
-    <View style={{ flex: 1, justifyContent: 'center', alignItems: 'center' }}>
-      <Text>Information</Text>
-    </View>
+    <FlatList
+      data={cardsData}
+      renderItem={renderItem}
+      keyExtractor={(item) => JSON.stringify(item)}
```

Comment:
```suggestion
      keyExtractor={item => item.title}
```
Could be simplified a bit